src/DIRAC/Core/Tornado/Server/private/BaseRequestHandler.py

```diff
@@ -31,24 +30,80 @@
 from DIRAC.Core.Security.X509Chain import X509Chain  # pylint: disable=import-error
 from DIRAC.FrameworkSystem.Client.MonitoringClient import MonitoringClient
 from DIRAC.Resources.IdProvider.Utilities import getProvidersForInstance
-
 from DIRAC.Resources.IdProvider.IdProviderFactory import IdProviderFactory
 
 sLog = gLogger.getSubLogger(__name__.split(".")[-1])
 
 
-class TornadoResponse(object):
-    """This class registers tornadoes with arguments in the order they are called
-    from TornadoResponse to call them later.
+def SETTINGS(attr, val):
```

Comment:
Normally this should be `settings` (lowercase) but maybe a better name could be used?

Reply:
I made them from the uppercase so that they look like the names of the default handler values:
`DEFAULT_AUTHENTICATION` -- `@AUTHENTICATION`
`DEFAULT_AUTHORIZATION`   -- `@AUTHORIZATION`
`DEFAULT_LOCATION`              -- `@LOCATION`
```
class MyAPI(TornadoREST):
    DEFAULT_LOCATION = '/api'
    
    def post_echo(self):
        # POST /api/echo
        pass
        
    @LOCATION('/some/path')
    def get_echo(self):
        # GET /some/path
        pass
```
But you right function should be lowercase, so what you prefer?

In a fact it's "method attribute setter", how to name it shortly.. set_attribute will be OK?

```
@set_attribute('my_attr', 'my_val')
def post_echo(self):
   pass
```